### ISSUE TITLE
APK

### ISSUE THREAD

**Issue by ryan-foamspace** [dhodge229/Sustain-Web3-hackathon-FOAM#1]
Thanks for your submission! Can you give permission to the APK link so we can install and test?

**Comment by dh06IRL** [comment]
Sorry about that, thought I made the correct permission changes!
Give it a shot now. https://drive.google.com/file/d/1lcKpvQED1lsdzTQyL5CiSYUxW2fe_2Ft/view?usp=sharing

Updating the gitignore to allow for APKs and pushing the same APK to this repo just incase. 

Extra note, it's a debug build that uses main net / production. Can adjust that if needed. May also pull down a heavy amount of POIs when in NYC, so can adjust filter settings / query results as well.

**Comment by ryan-foamspace** [comment]
@dhodge229 was not able to install with above link see error message below 
<img width="431" alt="Screen Shot 2020-02-13 at 5 21 33 PM" src="https://user-images.githubusercontent.com/18092788/74483996-b1461b00-4e85-11ea-8e6f-eaae85baa4d1.png">

**Comment by dh06IRL** [comment]
Not too sure what's causing that, but just pushed a signed APK up. Double checked and tested on a fresh device.

https://github.com/dhodge229/Sustain-Web3-hackathon-FOAM/blob/master/app/release/app-release.apk

**Comment by dh06IRL** [comment]
Extra note, the core AR functionality requires an ARCore supported device. Shouldn't cause install issues, but would in-app if not supported. https://developers.google.com/ar/discover/supported-devices

**Comment by ryan-foamspace** [comment]
@dhodge229 great thanks! Was able to install and load. Really Cool work!!

For us when testing the screen is first overloaded with PoIs and then they all disappear. Is this intended behavior?

I think it should show the closes PoI instead of everyone in that direction, check out image below for reference. 

![Screenshot_20200214-162809](https://user-images.githubusercontent.com/18092788/74569056-80c6b580-4f47-11ea-8694-cb8129758925.png)

**Comment by dh06IRL** [comment]
Yeah that's somewhat anticipated behavior. The tricky piece is the wide range of POI density in various areas. Where I am (and tested) there were only 2 within ~25mi radius.

That said, it's a quick change to adjust the render distance (how close a POI needs to be to b rendered) along with the query radius of how many are loaded. 

The readme included some items around that specific topic of how it could (should) be further optimized. A mix of logic handling rendering density, as well as some user preferences feels like a high level 'best fit'. 

Extra note, landscape wasn't entirely taken into account here. Would need to rotate the POI renderings in the AR view to accommodate various rotations.

**Comment by ryan-foamspace** [comment]
@dhodge229 good to know! As mentioned re: disappearing POIs from view- landscape helped 'see them' a bit better.

**Comment by dh06IRL** [comment]
@ryan-foamspace Just pushed a few adjustments for examples sake of adjusting rendering logic
https://github.com/dhodge229/Sustain-Web3-hackathon-FOAM/commit/c8371af25097cc4ccf355c906589c7f06bb1499e

New APK: https://github.com/dhodge229/Sustain-Web3-hackathon-FOAM/blob/master/app/release/app-release.apk

**Comment by XYONY** [comment]
Hey Dave, do you have a test flight code release for iOS yet?
